FreeRTOS/FreeRTOS-Kernel
# Add git attributes

- Adds `.gitattributes` that is configured to normalize line-endings to LF at checkin, for the files it deems "text"
- Replaces CRLF with LF for other files in changelist besides `.gitattributes`

Test Steps
-----------
Check in modifications to a "text" file with CRLF. See `.gitattributes` for set of "text" files. Upon committing changes to said file, Git will warn you of impending replacement of CRLF to LF upon push. 

Related Issue
-----------
[FreeRTOS-Kernel#234](https://github.com/FreeRTOS/FreeRTOS-Kernel/issues/234)


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
